platform/raspberry_pi/cmds/dma.c

```diff
@@ -0,0 +1,65 @@
+#include <drivers/bcm283x_dma.h>
+#include <unistd.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <mem/sysmalloc.h>
+#include <assert.h>
+
+#define MEM_SIZE 1024
+#define CPY_BLK 256
+#define DMA_CHAN 5
+
+void fill_test(uint32_t *memLoc, uint32_t bytes) {
```

Comment:
please make all internal routines static. It is useful to see use the routine only in this C-file or not